llvm/lib/MC/MCDwarf.cpp

```diff
@@ -158,8 +168,12 @@ void MCLineSection::addEndEntry(MCSymbol *EndLabel) {
   if (I != MCLineDivisions.end()) {
     auto &Entries = I->second;
     auto EndEntry = Entries.back();
-    EndEntry.setEndLabel(EndLabel);
-    Entries.push_back(EndEntry);
+    // If generatingFuncLineTableOffsets is set, then we already generated an
```

Comment:
Can you please move this comment up to be the third case? So change line 161 to say

// There are three cases:

and then add this comment as the third case?